site/src/pages/components/accordion.explainer.mdx

```diff
@@ -154,6 +174,12 @@ If this is the case,
 it seems likely that such changes
 may also be desirable for non-exclusive accordions (see next question).
 
+Based on the [discussion on 2023 April 27](https://github.com/openui/open-ui/issues/725#issuecomment-1525924478),
+it seems like at least default keyboard behavior for accordions
+is undesirable because it leads to bad user experience
+particularly for users on small or highly-zoomed displays
+(where the content of a single item is less likely to fit within the display).
```

Comment:
...or perhaps it leads to the need for some other feature (within focusgroup?) that allows the sub-items to be accessibly scrolled also? I don't know.